package.json

```diff
@@ -256,14 +257,14 @@
     "@lavamoat/lavadome-react": "0.0.17",
     "@lavamoat/snow": "^2.0.1",
     "@material-ui/core": "^4.11.0",
-    "@metamask-institutional/custody-controller": "^0.2.24",
-    "@metamask-institutional/custody-keyring": "^1.0.12",
-    "@metamask-institutional/extension": "^0.3.20",
-    "@metamask-institutional/institutional-features": "^1.2.15",
+    "@metamask-institutional/custody-controller": "^0.2.23",
```

Comment:
It looks like there are a number of MMI-related changes here - these packages are being downgraded. Is this intentional? It doesn't appear connected to the PR

Reply:
that is very weird, i look at the change, those 259 to 261 was added yesterday to `develop` branch , i will click `update branch` in this PR again to see any changes in `package.json`, if still have some library downgraded by `update branch` i will manually apply to `develop` branch version of those libraries.